LeaderboardBackend.Test/Controllers/UsersControllerTests.cs

```diff
@@ -62,9 +61,9 @@ public async Task GetUser_Ok_UserExists()
 			.Setup(x => x.GetUser(defaultUserId))
 			.Returns(Task.FromResult<User?>(defaultUser));
 
-		ActionResult<User> response = await _controller.GetUser(defaultUserId);
+		var response = await _controller.GetUser(defaultUserId);
+		var user = Helpers.GetValueFromObjectResult<OkObjectResult, User>(response);
```

Comment:
Same with these as above.

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d (`UsersControllerTests.cs` [`R60-61`](https://github.com/leaderboardsgg/leaderboard-backend-poc/commit/76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d#diff-13af0605f4348f02b5413ae0afb66abcd66be4608cf23ab7eba31ee049d667a6R60-R61)).